buku

```diff
@@ -3134,84 +3134,11 @@ n: don't add parent folder as tag
             index: Optional[int] = None,
             url: Optional[str] = None,
             shorten: bool = True) -> Optional[str]:
-        """Shorten a URL using Google URL shortener.
+        """Shorten/expand a URL using the tny.im service.
```

Comment:
Modified the original logic description, since it was factually incorrect (referring to a wrong service).